index.bs

```diff
@@ -4779,8 +4799,8 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 1. Let |preload script map| be |session|'s [=preload script map=].
 
-1. Set |preload script map|[|script|] to a struct with <code>expression</code>
-   |expression| and <code>sandbox</code> |sandbox|.
+1. Set |preload script map|[|script|] to a struct with <code>function
+   declaration</code> |function declaration|, , and <code>sandbox</code> |sandbox|.
```

Comment:
I think we want to use the `<code>` variant here, and please also remove the left over comma.

Reply:
I don't know what you mean by "the `<code>` variant".

Reply:
I've read that wrongly. Please ignore.